.github/workflows/macos.yml

```diff
@@ -63,6 +63,9 @@ jobs:
     - run: echo libtool autoconf automake pkg-config ${{ matrix.build.install }} | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
       name: 'brew bundle'
 
+    - run: brew uninstall openssl@1.0.2t && brew uninstall python@2.7.17 && brew untap local/openssl && brew untap local/python2 && brew install openssl@1.1
```

Comment:
Isn't this redundant to the version specified in install above?

Reply:
redundant how? If you can improve this, please be my guest. It took me almost a dozen iterations to get this to work yday... =)

Reply:
I will give it a try, it _should_ be possible to achieve this from within the generated Brewfile.